.devcontainer/devcontainer.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Retrospective Extension Development Environment",
-  "dockerFile": "Dockerfile",
+  "dockerFile": "Dockerfile.dev",

```

Comment:
I wonder if we need separate docker file just for jest? can we merge dockerfile and dockerfile.dev? so, dev environment will have all the tools that need to develop, test and publish the project?

Reply:
I am wondering if there will be any dependencies in the future that are specific to the environments that'll make the size of the containers different. Jest itself is relatively small, but I was about future cases.

But this might ultimately be over-engineering, I'm realizing, so I'll just add jest to the original Dockerfile.